script/run-container.sh

```diff
@@ -160,6 +160,10 @@ if [ -n "${DNS_ARG}" ]; then
 	DNS_ARG="--dns $DNS_ARG"
 fi
 
+if [ -n "${TNF_ENABLE_CRC_TESTING}" ]; then
+	ADD_HOST_ARG="--add-host api.crc.testing:host-gateway"
+fi
+
```

Comment:
```suggestion
ADD_HOST_ARG="${TNF_ENABLE_CRC_TESTING:+--add-host api.crc.testing:host-gateway}"

```

`ADD_HOST_ARG` will be assigned `--add...` value if `TNF_ENABLE_CRC_TESTING` is set, otherwise `''`. With current state `$ADD_HOST_ARG` fails when executed with `set -o nounset`, which is recommended. This can be addressed in a separate pull request for all variables.